File: python/34a_707A_BrainsPhotos.py
def main():
    flag = 0
    n, m = input().split()
    n, m = int(n), int(m)
    for i in range(n):
        row_pix = input().split()
        for j in row_pix:
            if j != "W" and j != "B" and j != "G":
                flag = 1
    if flag == 0:
        print("#Black&White")
    else:
        print("#Color")

if __name__ == "__main__":
    main()

File: cpp/34a_707A_BrainsPhotos.cpp
#include<bits/stdc++.h>

int main(){
    int n, m, flag = 0;
    char pix;
    std::cin>>n>>m;
    for(int i = 0; i < n; i++){
        for(int j = 0; j < m; j++){
            std::cin>>pix;
            if(pix != 'B' && pix != 'W' && pix != 'G'){
                flag = 1;
            }
        }
    }
    if(flag == 0){
        std::cout<<"#Black&White";
    }
    else{
        std::cout<<"#Color";
    }
    return 0;
}